custom_components/trakt_tv/apis/trakt.py

```diff
@@ -390,6 +390,59 @@ async def fetch_recommendations(self, configured_kinds: list[TraktKind]):
 
         return res
 
+    # start of new code
+    async def fetch_anticipated(self, path: str, limit: int):
+        return await self.request(
+            "get", f"{path}/anticipated?limit={limit}&ignore_collected=false"
+        )
+
+    async def fetch_anticipated_movies(self, limit: int):
+        return await self.fetch_anticipated("movies", limit)
+
+    async def fetch_anticipated_shows(self, limit: int):
+        return await self.fetch_anticipated("shows", limit)
+
```

Comment:
You can keep only fetch_anticipated and instead pass the path from KIND:
```
    async def fetch_anticipated(self, path: str, limit: int):
        return await self.request(
            "get", f"{path}?limit={limit}&ignore_collected=false"
        )
````

Reply:
New fetch_anticipated looks good, but remove fetch_anticipated_movies and fetch_anticipated_shows, you're not using them anymore